Stripe/Info.plist

```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>Stripe</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.stripe.stripe-ios</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
yep – it's in our project xcconfig: https://github.com/stripe/stripe-ios/blob/a26ff49ddfbcee9b1cc7e82b1ecfdd66794d09f4/Stripe/BuildConfigurations/Project-Shared.xcconfig#L139